### PR TITLE
feature/eng-2344-clustered-routing-template-cluster

### PR DESCRIPTION
**Pull request by nilsbeck**
# Description

This PR adds a new cluster-tsp template.

## Changes

Adds the project file to be used by the `nextmv init -t cluster-tsp` command.

### CODE REVIEW

File: templates/cloud-routing/README.md [hunk 1]
```diff
@@ -12,8 +12,7 @@ default the workspace file points to `fleet-tiny.json`. In addition there is the
 which helper functions are defined, e.g. for data handling.
 
 `main.go` is the entry point for the VRP solver. The actual configuration can be
-found in the `routing` package of this template. In particular
-`routing/solver.go`.
+found in particular `solver.go`.
```
Comment (nmisek) [inline templates/cloud-routing/README.md]:
```suggestion
found in `solver.go`.
```

File: .golangci.yml [hunk 2]
```diff
@@ -222,6 +222,15 @@ issues:
...
         - gocyclo
+    - path: templates/cluster-tsp/helper\.go
+      linters:
+        - gocyclo
+    - path: templates/cluster-tsp/schema\.go
+      linters:
+        - gocyclo
+    - path: templates/cluster-tsp/solver\.go
+      linters:
+        - gocyclo
```
Comment (merschformann) [inline .golangci.yml:233]: I will hopefully get to setting different limits for `gocyclo`. :sweat_smile:

File: templates/cluster-tsp/input.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
+{
+  "vehicles": ["v1", "v2"],
+  "stops": [
```
Comment (merschformann) [inline templates/cluster-tsp/input.json:3]: The names of the stops seem a little strange, e.g., `West` vs. `Place 1`. However, it's not a blocker.

**Review by merschformann — APPROVED** [review]
:rocket:

**Review by nmisek — APPROVED** [review]

**Pull request merged**